kubetest/gke.go

```diff
@@ -102,6 +102,8 @@ type gkeDeployer struct {
 	setup          bool
 	kubecfg        string
 	instanceGroups []*ig
+	bastionIPName  string
```

Comment:
What is bastionIPName? What is IPName?

Reply:
I'm sorry, it was old version of the code (with static IPs). Somehow my next push landed in the wrong place.